crates/sargon/src/profile/v100/profile.rs

```diff
@@ -294,7 +294,7 @@ impl Profile {
             .items()
             .into_iter()
             .map(Into::<AccountOrPersona>::into)
-            .map(|e| (e.clone(), e.unique_factor_instances()))
+            .map(|e| (e.clone(), e.unique_tx_signing_factor_instances()))
```

Comment:
Why not all instances including authentication_signing?

Reply:
good catch, this should have been `unique_all_factor_instances`, thx, will change